README.md

```diff
@@ -30,6 +30,15 @@ Is available at
 
 ## Explanation of Provided Tasks
 
+### build_dependencies:list
+Scans the `*.spec` files and prints the found `BuildRequires` dependencies.
+This can be used to get the list of packages needed for development.
```

Comment:
it is not needed for development but for build of package. ( even there is often intersection )